web/Controllers/LandlordController.cs

```diff
@@ -127,29 +127,46 @@ public async Task<ActionResult> ApproveCharter(int landlordId)
         return RedirectToAction("Profile", "Landlord", new { Id = landlordId });
     }
 
+
     [HttpGet]
-    [Route("/properties")]
-    public IActionResult ViewProperties()
+    [Route("properties")]
+    [Route("{id:int}/properties")]

```

Comment:
Slight preference for the pattern used by `/profile`, i.e. have `/me/properties` and `{id:int}/properties`

Reply:
That's a good point, I've updated line 132 above to be me/properties

Reply:
I also changed the statuscode on not finding a user landlord ID to be a 404, instead of a 500, to bring it in line with the me/profile status code.